content/tutorials/network-local.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Ganache Local Testnet 
+description: Details of Ganache deployment & services
+---
+
+## Smart Contracts
+
+[Ganache](https://www.trufflesuite.com/ganache) is a local Ethereum "network" that you run on your machine for fast iterations while developing your app. 
+
+You can deploy Ocean contracts onto Ganache. To do so, please go to [Ocean contracts repo](https://www.github.com/oceanprotocol/contracts) and follow the instructions there.
+
+## Connect Barge to Ganache
```

Comment:
Barge is running ganache so this sounds a bit confusing. The main use case would be to run only barge which comes then with ganache, aquarius, provider.

Reply:
Made it much better with commit e5d8816..0a71aa4. 

If this is fine with you, please 'resolve conversation'